internal/viewmodel/message.go

```diff
@@ -1,21 +1,52 @@
 package viewmodel
 
+import (
+	"backend/internal/model"
+)
+
 type BaseMessageView struct {
-	TelegramMessageId int64     `json:"telegram_message_id"`
+	TelegramMessageID int64     `json:"telegramMessageId"`
 	By                string    `json:"by"`
-	MessageBody       string    `json:"message_body"`
+	MessageBody       string    `json:"messageBody"`
 	Timestamp         string    `json:"timestamp"`
-	HotelStaffId      *uint     `json:"hotel_staff_id,omitempty"`
-	HotelStaff        *UserView `json:"hotel_staff,omitempty"`
-	RequestQueryId    uint      `json:"request_query_id,omitempty"`
+	HotelStaffID      *uint     `json:"hotelStaffId,omitempty"`
+	HotelStaff        *UserView `json:"hotelStaff,omitempty"`
+	RequestQueryID    uint      `json:"requestQueryId,omitempty"`
 }
 
 type MessageView struct {
 	BaseMessageView
-	RequestQuery BaseRequestQueryView `json:"request_query"`
+	RequestQuery *BaseRequestQueryView `json:"requestQuery,omitempty"`
 }
 
 type MessageWebSocketView struct {
 	BaseMessageView
-	ChatID uint `json:"chat_id,omitempty"`
+	ChatID uint `json:"chatId,omitempty"`
+}
+
+func BaseMessageViewFrom(message *model.Message) BaseMessageView {
+
+	var hotelStaff *UserView
+	if message.HotelStaff != nil {
+		hotelStaff = UserViewFrom(message.HotelStaff)
+	} else {
+		hotelStaff = nil
+	}
+	return BaseMessageView{
+		TelegramMessageID: message.TelegramMessageID,
+		By:                string(message.By),
+		MessageBody:       message.MessageBody,
+		Timestamp:         message.Timestamp.Format("2006-01-02 15:04:05"),
```

Comment:
```suggestion
		Timestamp:         message.Timestamp.Format(time.DateTime),
```
See https://pkg.go.dev/time#pkg-constants